marketing/tasks.py

```diff
@@ -91,8 +97,51 @@ def send_followup_emails():
                 )
 
     # Try sending all the followup emails
-    for args in mails_to_send:
-        render_and_dispatch_email(*args)
+    try:
+        for args in mails_to_send:
+            template, subscriber = args
+            render_and_dispatch_email(template, subscriber)
+            sent_emails[template.name].add(subscriber.user.email)
+    finally:
+        send_report(sent_emails)
+
+
+@db_task(retry=3, retry_delay=10)
+def send_report(sent_emails: Dict[str, Set]):
+    """
+    Render and send the report email.
+
+    This task has to be called from send_followup_emails.
+
+    The report email will not be sent if either the sent_emails is an
+    empty dictionary or settings.MARKETING_EMAIL_REPORT_RECIPIENTS is
+    not set.
+    """
+    recipients = settings.MARKETING_EMAIL_REPORT_RECIPIENTS
+    if not sent_emails:
+        logger.info("No markerting emails were sent. Skipping report email.")
+        return
+    if not recipients:
+        logger.info("MARKETING_EMAIL_REPORT_RECIPIENTS setting not set. Skipping report email.")
+        return
+    context = {
+        'sent_emails': dict(sent_emails)
+    }
+    plaintext_body_template = get_template('email_report.txt')
+    html_body_template = get_template('email_report.html')
+    plaintext_body = plaintext_body_template.render(context)
+    html_body = html_body_template.render(context)
+
+    try:
+        send_mail(
+            subject="Daily report for marketing emails",
+            message=plaintext_body,
+            html_message=html_body,
+            from_email=settings.MARKETING_EMAIL_SENDER,
+            recipient_list=recipients
+        )
+    except smtplib.SMTPException:
+        logger.error("Failed to send the report for marketing emails!")
```

Comment:
So that we don't miss out on useful debug info. :)

```suggestion
    except smtplib.SMTPException as e:
        logger.exception("Failed to send the report for marketing emails! %s", e)
```